salt/modules/consul.py

```diff
@@ -101,7 +114,7 @@ def _query(function,
     return ret
 
 
-def list_(consul_url=None, key=None, **kwargs):
+def list_(consul_url=None, token=None, key=None, **kwargs):
```

Comment:
Can you move the addition of the `token` option after `key`? That way we can avoid an API change for other modules calling out to this function.